### PR TITLE
:sparkles: Add check for comments in PR description

### LINKED ISSUE #5: PR description check should ensure that comments are not present
We use some comments in the PR templates, and we should ensure that those don't make it into the final PR or commit message.  Plus, comments could be used to hide things that would show up in the text commit log, which is not great.

/good-first-issue

### PR DESCRIPTION
**Pull request by paranoidsp**
This adds a check for comments in the PR description.

Fixes #5.

### CODE REVIEW

**Comment by paranoidsp** [comment]
There do not currently seem to be any tests for the PRPlugins in the verify runner, please let me know if you'd like me to add some based on the tests in the notes package.

**Comment by paranoidsp** [comment]
/assign @directxman12

Please review whenever you get a chance.

File: verify/cmd/runner.go [hunk 1]
```diff
@@ -87,6 +87,10 @@ func main() {
 					errs = append(errs, "**Your PR has an issue number in the title.**\n\nThe title should just be descriptive.\nIssue numbers belong in the PR body as either `Fixes #XYZ` (if it closes the issue or PR), or something like `Related to #XYZ` (if it's just related).")
 				}
 
+				if regexp.MustCompile(`(?s)<!--.*?-->`).MatchString(pr.GetBody()) {
```
Comment (DirectXMan12) [inline verify/cmd/runner.go:90]: this'll get comments inside code blocks, and other stuff too.  At the very least we need to check https://github.com/russross/blackfriday/blob/v2.0.1/html.go#L68.

The "more correct" solution is probably to parse the markdown with blackfriday & look for html nodes that are comment nodes, I think.

**Review by DirectXMan12 — CHANGES_REQUESTED** [review]

**Comment by DirectXMan12** [comment]
(sorry for the delay in review)

**Comment by Adirio** [comment]
Should this be implemented as a new `PRPlugin`? This way we would achieve higher concurrency and more helpful error messages.

File: verify/cmd/runner.go (hunk 1 above)
Comment (Adirio) [reply verify/cmd/runner.go:90]: +1 for the "more correct" solution

**Review by Adirio — CHANGES_REQUESTED** [review]

**Comment by DirectXMan12** [comment]
> Should this be implemented as a new PRPlugin? This way we would achieve higher concurrency and more helpful error messages.

Yeah, esp if it's gonna be more complex

**Pull request closed without merging**